src/utils/filterVideo.js

```diff
@@ -0,0 +1,23 @@
+function filteredVideosOnCategory(selectedCategory, videos) {
+  let filterVideos = videos;
+
+  if (selectedCategory == "All") {
+    return filterVideos;
+  } else if (selectedCategory == "International Comics") {
+    filterVideos = filterVideos.filter(
+      (video) => video.popularity == "International"
+    );
+  } else if (selectedCategory == "Indian Comics") {
+    filterVideos = filterVideos.filter(
+      (video) => video.nationality == "Indian"
+    );
+  } else {
+    filterVideos = filterVideos.filter(
+      (video) => video.genre == selectedCategory
+    );
+  }
+
+  return filterVideos;
```

Comment:
Appreciation : Using one common return instead of doing it in each of the condition.